safedelete/tests/settings.py

```diff
@@ -34,6 +34,7 @@
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.admin',
+    'django.contrib.messages',
```

Comment:
Any reason why this was added?

Reply:
```
$ coverage run `which django-admin.py` test --settings=safedelete.tests.settings
Creating test database for alias 'default'...
Destroying test database for alias 'default'...
SystemCheckError: System check identified some issues:
ERRORS:
?: (admin.E406) 'django.contrib.messages' must be in INSTALLED_APPS in order to use the admin application.
System check identified 1 issue (0 silenced).
The command "coverage run `which django-admin.py` test --settings=safedelete.tests.settings" exited with 1.
Done. Your build exited with 1.
```

Django 2.2 and python 3.7 had failed tests because of this